tests/unit/services/test_lifecycle.py

```diff
@@ -51,23 +51,108 @@ def fake_parts_lifecycle(app_metadata, fake_project, tmp_path):
 
 # endregion
 # region Helper function tests
-@pytest.mark.parametrize("step", Step)
-@pytest.mark.parametrize("reason", [None, "Because I said so!"])
-def test_get_parts_action_message_run(step: Step, reason: str | None):
+@pytest.mark.parametrize(
+    ("step", "message"),
+    [
+        (Step.PULL, "Pulling my-part"),
+        (Step.BUILD, "Building my-part"),
+        (Step.OVERLAY, "Overlaying my-part"),
+        (Step.STAGE, "Staging my-part"),
+        (Step.PRIME, "Priming my-part"),
+    ],
+)
+def test_get_parts_action_message_run(step: Step, message: str):
     action = Action(
         "my-part",
         step,
         action_type=ActionType.RUN,
-        reason=reason,
     )
 
     actual = lifecycle._get_parts_action_message(action)
 
-    pytest_check.is_true(actual.startswith(f"{step.name.lower()} my-part"))
-    if reason:
-        pytest_check.is_true(actual.endswith(f"({reason})"))
-    else:
-        pytest_check.is_false(actual.endswith(")"))
+    assert actual == message, "Unexpected %r" % actual
+
+
+@pytest.mark.parametrize(
+    ("step", "message"),
+    [
+        (Step.PULL, "Repulling my-part"),
+        (Step.BUILD, "Rebuilding my-part"),
+        (Step.OVERLAY, "Re-overlaying my-part"),
+        (Step.STAGE, "Restaging my-part"),
+        (Step.PRIME, "Re-priming my-part"),
```

Comment:
opinion: removing the hyphen from `re-prime` would make the output more consistent, except rockcraft because `reoverlaying` doesn't look right.

Another option would be to hyphenate all 5 verbs.

Reply:
I'm not sure there is a good answer here - I think hyphenating everything would tend towards Emerson's "foolish consistency", so I prefer it either as-is or with the removal of the hyphen for priming.

Reply:
I have no objections to `reprime` and `re-overlay`.

Reply:
I'll make that change and figure out what happened to the testing